### DISCUSSION TITLE
Size App > 100MB

### DISCUSSION THREAD

**Discussion by fsotelos** [dotnet/maui#22627] [General]
Hi there!

I have a problem with the size of my app.

I am working with Firebase.

But exists one problem when I try to generate my Release, I put the property trimming publish. Then the login with firebase leaves for work.

I try resolve my problem in this forum 

https://www.syncfusion.com/forums/188198/size-of-app-100mb

Somebody help me!

Best Regards 

Felipe

**Comment by stefandb1995** [comment]
Possible due to AOT. It does increase the size of the APK by a fairly large amount. You could try and disable it? It's on by default for Release builds.

**Reply by fsotelos** [reply]
Hi @stefandb1995 

Actually I have disabled that feature

![image](https://github.com/dotnet/maui/assets/126302871/e5325a96-3c1f-45c7-abf6-34d43038c0dc)

**Comment by Pozi09** [comment]
Hi. I can't help you with your problem, but I'm great at promoting apps.
